diff -aurN Gnome2-Wnck-0.05.orig/xs/WnckWindow.xs Gnome2-Wnck-0.05/xs/WnckWindow.xs
--- Gnome2-Wnck-0.05.orig/xs/WnckWindow.xs	2004-07-10 16:25:31.000000000 +0200
+++ Gnome2-Wnck-0.05/xs/WnckWindow.xs	2004-07-26 00:04:16.017540584 +0200
@@ -141,7 +141,7 @@
 	WnckWindow *window
 	gboolean skip
 
-#if WNCK_CHECK_VERSION (2, 6, 3)
+#if WNCK_CHECK_VERSION (2, 6, 2)
 
 ##  void wnck_window_close (WnckWindow *window, guint32 timestamp) 
 void
